mbed_lstools/lstools_base.py

```diff
@@ -109,14 +109,14 @@ def list_mbeds_ext(self):
 
     def list_mbeds(
             self, fs_interaction=FSInteraction.BeforeFilter,
-            platform_name_filters=[], unique_names=False,
+            filter_function=None, unique_names=False,
```

Comment:
could we get also public API tests? :) anyway, this looks better approach 👍

Reply:
could be check that filter_function is really function when given

Reply:
Yeah, probably wouldn't hurt, that can be added in another PR.

Reply:
so API test for this is coming later, @theotherjimmy @bridadan ?